frontend/packages/client/src/App.sass

```diff
@@ -58,7 +58,7 @@ $tooltip-radius: 6px
 $popover-max-width: 20rem
 @import '@creativebulma/bulma-tooltip'
 @import 'bulma-popover'
-@import 'bulma-switch'
+@import '~bulma-switch'
```

Comment:
what does this `~` do here? do we need it on `bulma-popover` too?

Reply:
The `~` is following the documentation of bulma extensions
![Screen Shot 2022-10-19 at 10 57 43 AM](https://user-images.githubusercontent.com/7423845/196768824-2667c458-39c5-4dcc-b038-6e6a37efdee6.png)

I'll try to remove and see if it still works, bulma-popover is added before, not sure of its usage

Reply:
Removed!